src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/Nationality.java

```diff
@@ -0,0 +1,247 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+
+public enum Nationality {
+    AFGHAN,
```

Comment:
We need to bottom out [this](https://github.com/companieshouse/limited-partnerships-api/pull/82/files#r1965848152) discussion before deciding on this list, so it will be based on @dlloyd1-ch 's schema definition [here](https://github.com/companieshouse/limited-partnerships-api/pull/82).

Also not sure about the underscores here as they don't get sent through to CHIPS normally. E.g. for ROE we sent:

     `     <nationalityOther>NEW ZEALANDER</nationalityOther>`

Reply:
Using descriptions on the enum so that enum caps and underscore are not stored or passed along the code stack